toolchain/sem_ir/absolute_node_id.h

```diff
@@ -11,10 +11,49 @@
 
 namespace Carbon::SemIR {
 
-// A specific node location in a file.
-struct AbsoluteNodeId {
-  CheckIRId check_ir_id;
-  Parse::NodeId node_id;
+// A specific node location in a file. Can refer to a Clang source location
+// within imported C++ code.
+class AbsoluteNodeId {
+ public:
+  // A speicif cnode location in a file.
```

Comment:
was just browsing this PR and noticed a typo here

Reply:
Thanks! Done.